packages/dumb/src/components/trends/cbg/cbg-slice/cbg-slice-animated.util.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2017-2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import {
+  RANGE_SEGMENT_FIRST,
+  RANGE_SEGMENT_MAX,
+  RANGE_SEGMENT_NINETIETH,
+  RANGE_SEGMENT_TENTH,
+  RANGE_SEGMENT_THIRD
+} from '../../../../models/constants/cbg.constants'
+import { TrendsDisplayFlags } from '../../../../models/trends-display-flags.model'
+import { CbgRangeSegment } from '../../../../models/cbg-range-segment.model'
+
+export const getRangeSegments = (displayFlags: TrendsDisplayFlags): CbgRangeSegment[] => {
```

Comment:
I'm not sure how I feel about having an `util` file with the components.. What do you think of moving it in the `utils` folder?

Reply:
In dumb, test and css are both right next with the components. It seems fitting to have the util with them? Especially as the util is specific to the component